supernode/services/artworkregister/node.go

```diff
@@ -33,7 +33,7 @@ func (nodes Nodes) ByID(id string) *Node {
 func (nodes *Nodes) Remove(id string) {
 	for i, node := range *nodes {
 		if node.ID == id {
-			*nodes = append((*nodes)[:i], (*nodes)[:i+1]...)
```

Comment:
did u do test for it?

Reply:
I did test it. its adding duplicate node at the end instead remove node

Reply:
yeah, the refactored code is different, and i have unit tests for it, it works well, good catch